src/App.js

```diff
@@ -1,15 +1,18 @@
-import logo from './logo.svg';
 import './App.css';
-import header from './Components/Header/header.js';
 import main from './Components/Main/main.js';
 import { Route, Switch } from 'react-router-dom';
+import notFound from './Components/not found/notFound.js';
+import Header from './Components/Header/header.js';
 
 function App() {
   return (
     <div className="App">
-   <header />
-   <Switch>
-   </Switch>
+      <Header />
+      <Switch>
+        <Route path="/rgb/:red/:green/:blue" component={main} />
```

Comment:
This is working, but there is no path for when the page loads. For example, "/" will always be the first path that loads; without giving that path a component or redirecting, your app always starts with a "404 not found" in its current state. If you would like some advice on handling this, let me know!